python/ray/ml/utils/tensorflow_utils.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import tensorflow as tf
+
+
+def convert_pandas_to_tf_tensor(df: pd.DataFrame) -> tf.Tensor:
```

Comment:
This is a new function. Can you add unit test?

Reply:
There are unit tests in `test_tensorflow_utils.py`